verification/src/http_server/handlers/verification/sourcify/types.rs

```diff
@@ -0,0 +1,42 @@
+use serde::{Deserialize, Serialize};
+use std::collections::HashMap;
+
+// This struct is used as input for our endpoint and as
+// input for sourcify endpoint at the same time
+#[derive(Debug, Deserialize, Serialize, PartialEq)]
+pub struct SourcifyRequest {
+    pub address: String,
+    pub chain: String,
+    pub files: HashMap<String, String>,
+}
+
+// Definition of sourcify.dev API response
+// https://docs.sourcify.dev/docs/api/server/v1/verify/
+#[derive(Deserialize)]
+#[serde(untagged)]
+pub enum SourcifyVerifyResponse {
```

Comment:
We may may expose those types to `sourcify/mod.rs` without exposing them to others
```suggestion
pub(super) enum SourcifyVerifyResponse {
```